### ISSUE TITLE
global: same asset should not be loaded twice

### ISSUE THREAD

**Issue by SamiHiltunen** [inveniosoftware/invenio-assets#20]
There is an issue with loading assets multiple times. For example, if two templates load invenio_search_js, this will cause an error to be raised since the Angular app will be loaded twice. Assets should rather collect the requirements and then add them only once to the output.

We discussed with @jirikuncar the possbile solutions to this. There are two approaches at least. One is to make a better Bundle class that will check if the asset has already been declared and not give out the same asset twice. Other option would be to modify the assets extensions and handle the problem there.

Relevant line in the assets extension:
https://github.com/miracle2k/webassets/blob/master/src/webassets/ext/jinja2.py#L191

**Comment by lnielsen** [comment]
Superseded by #69
